### PR TITLE
ADEN-6446 Add missing adengadinfo values for featuredvideo

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Review by nandy-andy — APPROVED** [review]

**Comment by nandy-andy** [comment]
LGTM :+1: 

PHP unit tests failed but the changes weren't connected to PHP - it's caused by the message cache rebuilt. To double-check run the tests on your devbox with the newest version of `Wikia/config` and paste here results, please ;)

File: extensions/wikia/AdEngine/js/tracking/adInfoListener.js [hunk 1]
```diff
@@ -90,10 +91,12 @@ define('ext.wikia.adEngine.tracking.adInfoListener',  [
 			slotPrices.a9 = vastInfo.amznbid;
 		}
 
+		slotRegistry.storeScrollY(vastInfo.pos);
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/tracking/adInfoListener.js]: We use this value for analysis around better targeting of ads. It means that the value should be calculated before we perform DFP request. It means that we should change how and when we call this method for video.

I'd suggest to call this method from https://github.com/Wikia/app/blob/fc4e1e880fcd072f66959723f32e76b114455d2f/extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js [hunk 2]
```diff
@@ -5,8 +5,9 @@ define('wikia.articleVideo.featuredVideo.ads', [
 	'ext.wikia.adEngine.slot.service.srcProvider',
 	'ext.wikia.adEngine.video.player.jwplayer.adsTracking',
 	'ext.wikia.adEngine.video.vastDebugger',
+	'ext.wikia.adEngine.slot.service.slotRegistry',
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js]: please try to keep alphabetical order in dependencies list when possible ;)

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js]: Done :)

File: extensions/wikia/AdEngine/js/tracking/adInfoListener.js [hunk 3]
```diff
@@ -91,9 +91,9 @@ define('ext.wikia.adEngine.tracking.adInfoListener',  [
 		}
 
 		tracker.track(
-			vastInfo.pos,
-			vastInfo,
-			vastInfo,
+			vastInfo.customParams.pos,
+			vastInfo.customParams,
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/tracking/adInfoListener.js:95]: Does line item id / creative id / media type / etc (all info about played ad video) still work after this change?

Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/tracking/adInfoListener.js:95]: I believe that this argument (second) should stay `vastInfo`. Please check some test page with targeted preroll:
http://project43.wikia.com/wiki/SyntheticTests/Premium/FeaturedVideo

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/tracking/adInfoListener.js:95]: Yep, everything is fine. This var (slotName) is used only for getting scroll_y from array.

Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/tracking/adInfoListener.js:95]: Oh, got it. Sorry!

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**